resources/lang/es/sms.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+return [
+    /*
+    |--------------------------------------------------------------------------
+    | SMS Language Lines
+    |--------------------------------------------------------------------------
+    */
+
+    // TODO: Get these translated
+    'subscribed'        => '¡Gracias por suscribirte! Envía un mensaje de texto para cancelar la suscripción. ¿Necesita registrarse para votar? ¡Vaya a https://bit.ly/voteictregister para hacerlo ahora, directamente desde su teléfono!',
```

Comment:
I think this says "send a text message to unsubscribe". It should it be more specific, something like "Texto DETENER para cancelar la suscripción" or "Texto DETENER para darse de baja" I'm not sure which is correct though

Reply:
I have a request out to Claudia to get better translations for all this stuff, so I'm going to punt for now and wait until those come in. Mostly because I'm on mobile right now and I want to merge this thing anyways :) lol